prerendering.bs

```diff
@@ -658,6 +673,37 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
+
+[=prerendering browsing context/Prerender scoped Accept-CH cache=] is an [=ordered map=] of [=origin=] to [=client hints set=].
+
+<div algorithm="add a new prerender Accept-CH cache entry">
+
+Similar to [=add a new Accept-CH cache entry=], to <dfn>add a new prerender Accept-CH cache entry</dfn> to the [=prerendering browsing context/prerender scoped Accept-CH cache=],
+given an |origin| and a [=client hints set=] |hintSet|,
+[=map/set=] [=prerendering browsing context/prerender scoped Accept-CH cache=][|origin|] to |hintSet|.
```

Comment:
The prerender scoped Accept-CH cache belongs to a given prerendering browsing context. So it needs to be referenced as `|bc|'s [=...=]`. Which probably means this algorithm needs to take a prerendering browsing context.

Reply:
Indeed. 👍  Fixed.